doc/interop-test-descriptions.md

```diff
@@ -392,22 +392,24 @@ Asserts:
 * clients are free to assert that the response payload body contents are zero
   and comparing the entire response message against a golden response
 
-### Metadata (TODO: fix name)
+### custom_metadata
 
-Status: Not yet implementable
+Status: Ready for implementation, beta requirement.
```

Comment:
That is different than we've done in the past. We've tracked implementation status separately. Is there a reason to doing that differently now? If so, then multiple pre-existing tests need to a status added because they aren't implemented in all languages.